shared/constants/swaps.ts

```diff
@@ -116,6 +120,7 @@ const POLYGON_CONTRACT_ADDRESS = '0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31';
 const AVALANCHE_CONTRACT_ADDRESS = '0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31';
 const OPTIMISM_CONTRACT_ADDRESS = '0x9dda6ef3d919c9bc8885d5560999a3640431e8e6';
 const ARBITRUM_CONTRACT_ADDRESS = '0x9dda6ef3d919c9bc8885d5560999a3640431e8e6';
+const ZKSYNC_CONTRACT_ADDRESS = '0xf504c1fe13d14df615e66dcd0abf39e60c697f34';
```

Comment:
What is this contract? It is not verified, doesn't look like a safe destination for Swaps

Reply:
When possible, we try to maintain address consistency when deploying the Swaps contract to new networks, but we were unable to do so in this instance.

Reply:
But can't it be verified?

Reply:
Swap contracts deployed by MetaMask are rigorously tested to ensure adherence to best practices & industry standards. Additionally, we intend to verify the contract as we approach a public release. Thank you for your attention to detail & for helping to keep MetaMask safe.